libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationViewModel.java

```diff
@@ -401,20 +402,20 @@ private void initializeNavigation(Context context, NavigationOptions options, Lo
 
   private void addNavigationListeners() {
     navigation.registerRouteProgressObserver(navigationViewVm);
-    navigation.unregisterLocationObserver(navigationViewVm);
-    navigation.registerOffRouteObserver(offRouteListener);
+    navigation.registerLocationObserver(navigationViewVm);
```

Comment:
`navigationProgressObserver` sounds a better name? And with this rename I will rename the `listener` to `NavigationViewModelProgressObserver` in the new UI SDK since we are naming others as `observer`